packages/web/modals/pre-transfer.tsx

```diff
@@ -41,16 +42,18 @@ export const PreTransferModal: FunctionComponent<
   const { t } = useTranslation();
 
   const tokenValue = priceStore.calculatePrice(
```

Comment:
Thoughts on using `useCoinFiatValue` instead of price store here? Would be nice, going forward, to remove all refs to price/query stores.

Reply:
I was not aware this hook existed. I have added this on my latest commit. Thank you!